src/number_format.js

```diff
@@ -277,8 +278,16 @@ class NumberFormat extends React.Component {
 
     //in case of format as number limit between prefix and suffix
     if (!format) {
-      const hasNegation = value[0] === '-';
-      return clamp(caretPos, prefix.length + (hasNegation ? 1 : 0), value.length - suffix.length);
+      let hasNegation;
+      let negationLength = 0;
+      if (accounting) {
+        hasNegation = value[0] === '(';
```

Comment:
We should also check if the number ends with ).

Reply:
Good point